docs/stories/from-sphinx-to-mkdocs.md

```diff
@@ -182,13 +205,13 @@ files into markdown code snippets.
 
 While there are at least two solutions for including content with [mkdocs], they
 both are problematic because some decisions that were made regarding how
-relative paths are computed, something that breaks badly when you try to build a
+relative paths are computer, something that breaks badly when you try to build a
```

Comment:
Heh, automatic spelling fixes? I reviewed them last time, and they went back to the previous, wrong version x)